qiskit_sphinx_theme/layout.html

```diff
@@ -267,116 +146,23 @@
-
-  <div>
-    <br>
-  </div>
-
-  <!-- Begin Mobile Menu -->
-
-  <div class="mobile-main-menu">
-    <div class="container-fluid">
-      <div class="container">
-        <div class="mobile-main-menu-header-container">
-          <a class="header-logo" href="{{ theme_variables.external_urls['home'] }}" aria-label="PyTorch"></a>
-          <a class="main-menu-close-button" href="#" data-behavior="close-mobile-menu"></a>
-        </div>
-      </div>
-    </div>
-
-    <div class="mobile-main-menu-links-container">
-      <div class="main-menu">
-        <ul>
-          <li>
-            <a href="{{ pathto('getting_started') }}">Getting Started</a>
-          </li>
-          <li>
-            <a href="{{ pathto('tutorials') }}">Tutorials</a>
-          </li>
-          <li>
-            <a href="{{ theme_variables.external_urls['partners'] }}">Partners</a>
-          </li>
-          <br>
-          <li class="resources-mobile-menu-title">
-            Applications
-          </li>
-          <ul class="resources-mobile-menu-items">
-            <li>
-              <a href="{{ theme_variables.external_urls['ml'] }}">Machine learning</a>
-            </li>
-
-            <li>
-              <a href="{{ theme_variables.external_urls['nature'] }}">Nature</a>
-            </li>
-
-            <li>
-              <a href="{{ theme_variables.external_urls['finance'] }}">Finance</a>
-            </li>
-
-            <li>
-              <a href="{{ theme_variables.external_urls['optim'] }}">Optimization</a>
-            </li>
-          </ul>
-          <br>
-          <li>
-            <a href="{{ theme_variables.external_urls['experiments'] }}">Experiments</a>
-          </li>
-          <br>
-          <li class="resources-mobile-menu-title">
-            Resources
-          </li>
-
-          <ul class="resources-mobile-menu-items">
-            <li>
-              <a href="{{ theme_variables.external_urls['slack'] }}">Slack support</a>
-            </li>
-          </ul>
-          <br>
-        </ul>
-      </div>
-    </div>
-  </div>
-
-  <!-- End Mobile Menu -->
+  
+  <!-- MOBILE MENU FOR SIDEBAR -->
+    <script type="text/javascript" id="documentation_options" data-url_root="{{ pathto('', 1) }}" src="{{ pathto('_static/documentation_options.js', 1) }}"></script>
+    {%- for scriptfile in script_files %}
+      {{ js_tag(scriptfile) }}
+    {%- endfor %}
+
+    <script type="text/javascript" src="{{ pathto('_static/js/vendor/popper.min.js', 1) }}"></script>
+    <script type="text/javascript" src="{{ pathto('_static/js/vendor/bootstrap.min.js', 1) }}"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.min.js"></script>
+    <script type="text/javascript" src="{{ pathto('_static/js/theme.js', 1) }}"></script>
+
+    <script type="text/javascript">
+        jQuery(function () {
+            SphinxRtdTheme.Navigation.enable({{ 'true' if theme_sticky_navigation|tobool else 'false' }});
```

Comment:
I asked during midsprint meeting. Do you know where the `SphinxRtdTheme` class/variable come from? I think we are not using sphinx-rtd-theme so we probably don't need this or it can be renamed where it's initialized.

Reply:
I tried removing this and it broke functionality in the sidebar. Im not sure exactly what this does and haven't had time to investigate properly. For now i think best strategy is to keep it.

Reply:
@javabster I did a search. The only place I could find this variable is in the `theme.js`. Can you try renaming this variable in `theme.js` and `layout.html` to `QiskitSphinxTheme` and check if that works? https://github.com/Qiskit/qiskit_sphinx_theme/blob/1035b33351150d640484e538e0123ca3c7d0da47/qiskit_sphinx_theme/static/js/theme.js#L878

Reply:
ok so i tried changing to `QiskitSphinxTheme` and it still worked, and then i tried removing it completely and it still worked, so maybe something else was breaking it when i checked last time. So i decided just to remove this script entirely 😄